src/topp/DatabaseSuitability.cpp

```diff
@@ -135,17 +134,17 @@ class DatabaseSuitability :
   // it gets automatically called on tool execution
   void registerOptionsAndFlags_() override
   {
-    registerInputFile_("in_id", "<file>", "", "Input idXML file from peptide search with combined database with added de novo peptide (after FDR)");
+    registerInputFile_("in_id", "<file>", "", "Input idXML file from peptide search with combined database with added de novo peptide. PeptideIndexer is needed, FDR is forbidden.");
     setValidFormats_("in_id", { "idXML" });
     registerInputFile_("in_spec", "<file>", "", "Input MzML file used for the peptide identification");
     setValidFormats_("in_spec", { "mzML" });
-    registerInputFile_("in_novo", "<file>", "", "Input idXML file containing de novo peptides");
+    registerInputFile_("in_novo", "<file>", "", "Input idXML file containing de novo peptides (unfiltered)");
     setValidFormats_("in_novo", { "idXML" });
     registerOutputFile_("out", "<file>", "", "Optional tsv output containing database suitability information as well as spectral quality.", false);
     setValidFormats_("out", { "tsv" });
-    registerDoubleOption_("novor_fract", "<double>", 1, "Set the fraction of how many cases, where a de novo peptide scores just higher than the database peptide, you wish to re-rank.", false, true);
-    setMinFloat_("novor_fract", 0);
-    setMaxFloat_("novor_fract", 1);
+    registerDoubleOption_("cut_off_fract", "<double>", 1, "Percentile to determine which decoy cut-off to use. '1' use the highest cut-off to '0' use the lowest one.", false, true);
```

Comment:
this is the `reranking_cutoff_percentile` (or along those lines)?

`Swap a top-scoring deNovo hit with a lower scoring DB hit if their xcorr score difference is less than threshold 't'.
't' is determined as the percentile (this parameter) in the list of all sorted score differences between two decoy hits of a PSM.` ??
.... phew...

Reply:
`Swap a top-scoring deNovo hit with a lower scoring DB hit if their xcorr score difference is in the given percentile of all score differences between the first two decoy hits of a PSM.`

Reply:
Percentile implies sorted.
And it should be clear that it is about this param. Especially if we rename to ..._percentile.